index.js

```diff
@@ -0,0 +1,5 @@
+// import server.js
+const server = require('./server');
+
+// set up server on port 5000
+server.listen(5000, () => console.log('Server is running on port 5000')); 
```

Comment:
it'd probably be better to use a variable for your port here so that it'd be easier to change:
```javascript
const port = 5000;
server.listen(port, () => console.log(`Server is running on port ${port}`));
```